core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2104,4 +2104,34 @@ private CarbonCommonConstants() {
    */
   public static final String CARBON_QUERY_DATAMAP_BLOOM_CACHE_SIZE_DEFAULT_VAL = "512";
 
+  /**
+   * The IP on which Index Server will be started.
+   */
+  @CarbonProperty
+  public static final String CARBON_INDEX_SERVER_IP = "carbon.index.server.ip";
```

Comment:
`CarbonCommonConstants.java` file has too many constants defined. Can we think of seggregating the properties based on their usage. Like all properties defined for index cache server can be moved to a separate constants file (e.g CacheIndexServerConstants) OR we can also think of introducing builder pattern to construct properties with default/specified values

Reply:
Yeah, we can do this as a seperate task